### ISSUE TITLE
Can't print node fs module

### ISSUE THREAD

**Issue by futuro** [anmonteiro/lumo#203]
In release 1.6.0, when you try to print the node Filesystem module with cljs printers, it throws the following error.

```Clojure
cljs.user=> (js/require "fs")
Cannot read property 'cljs$lang$ctorStr' of undefined
         Object.cljs.core.pr_writer_impl (NO_SOURCE_FILE <embedded>:1703:280)
         cljs.core.pr_writer (NO_SOURCE_FILE <embedded>:1705:415)
         (NO_SOURCE_FILE <embedded>:1725:340)
         Object.cljs.core.pr_sequential_writer (NO_SOURCE_FILE <embedded>:1692:265)
         Object.cljs.core.print_prefix_map (NO_SOURCE_FILE <embedded>:1725:65)
         Object.cljs.core.print_map (NO_SOURCE_FILE <embedded>:1726:359)
         Object.cljs.core.pr_writer_impl (NO_SOURCE_FILE <embedded>:1700:126)
         Object.cljs.core.pr_writer (NO_SOURCE_FILE <embedded>:1705:415)
         Object.cljs.core.pr_seq_writer (NO_SOURCE_FILE <embedded>:1706:51)
         Object.cljs.core.pr_sb_with_opts (NO_SOURCE_FILE <embedded>:1707:156)

cljs.user=>
```

Here is another test that throws the same error

```Clojure
(.. js/Object (entries fs) (forEach (fn [[key value] index arr] (println value))))
```
`(println keys` works fine here, and `(js/console.log (js/require "fs"))` works as well.

**Comment by anmonteiro** [comment]
this is fixed in CLJS master

**Comment by futuro** [comment]
@anmonteiro do you know what commit fixes it/how I might go about building a version of Lumo with the fix?

**Comment by anmonteiro** [comment]
fixed in #208
